addons/makeslave.pm

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/perl
```

Comment:
Our usual standard is to have scripts be `.pl` and perl modules `.pm`

Also that ensures the packaging picks it up automagically in addons/